exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorkerThread.java

```diff
@@ -113,13 +116,24 @@ void export() {
       registerView(view);
       viewDataList.add(viewManager.getView(view.getName()));
     }
-    CreateTimeSeriesRequest.Builder builder =
-        CreateTimeSeriesRequest.newBuilder().setNameWithProjectName(projectName);
+    List<TimeSeries> timeSeriesList = Lists.newArrayList();
     for (ViewData viewData : viewDataList) {
-      builder.addAllTimeSeries(StackdriverExportUtils.createTimeSeriesList(viewData, projectId));
+      timeSeriesList.addAll(StackdriverExportUtils.createTimeSeriesList(viewData, projectId));
     }
-    if (!builder.getTimeSeriesList().isEmpty()) {
-      metricServiceClient.createTimeSeries(builder.build());
+
+    for (List<TimeSeries> batchedTimeSeries :
+        Lists.partition(timeSeriesList, MAX_BATCH_EXPORT_SIZE)) {
+      // Batch export 3 TimeSeries at one call, to avoid exceeding RPC header size limit.
```

Comment:
Is there a relevant GitHub issue, discussion on SO, etc.? If so, it would be great to add a reference here.
Otherwise, LGTM.

Reply:
@bogdandrutu caught this bug in an E2E test (https://github.com/bogdandrutu/java-demo-apps). I've just created an issue for this: https://github.com/census-instrumentation/opencensus-java/issues/840.